docs/platforms/javascript/common/opentelemetry/custom-setup.mdx

```diff
@@ -111,7 +111,19 @@ If you want to add your own http/node-fetch instrumentation, you have to follow
 
 ### Custom HTTP Instrumentation
```

Comment:
Above there is also this sentence:

> These are needed to make sure that trace propagation works correctly. Additionally, the HTTP instrumentation is configured to ensure that request isolation is automatically applied for Sentry.

Which we should also adjust (I guess just remove the second sentence?)

Reply:
Right, good point. We no longer configure HttpInstrumentation for this but use our own instrumentation. Will remove the sentence